app/views/pages/our_story.html.erb

```diff
@@ -46,7 +46,7 @@
       <p>Existing community information websites were either maintained on behalf of the groups by institutional staff or required community groups to regularly log in to keep their events updated.</p>
     </div>
     <div class="reveal__body">
-      <p>This meant that community groups had to input their information in one site for each provider, in addition to social media sites like Facebook, Twitter and Instagram, creating an ever-increasing amount of work for a very small amount of publicity.</p>
+      <p>This meant that community groups had to input their information in one site for each provider, in addition to social media sites like Twitter and Instagram, creating an ever-increasing amount of work for a very small amount of publicity.</p>
```

Comment:
is this really necessary? :) We have a plan in progress to support Facebook, but also this text is talking about in retrospect, and we _did_ support Facebook.